### DISCUSSION TITLE
[ERROR][com.freerdp.client.x11] - Please check that the $DISPLAY environment variable is properly set

### DISCUSSION THREAD

**Discussion by nizarhamood** [FreeRDP/FreeRDP#8079] [Q&A]
Below is the error I get after installing FreerDP:

**_[ERROR][com.freerdp.client.x11] - Please check that the $DISPLAY environment variable is properly set_**

I've tried to reboot and use it again but the same error appears.

Looked on the net but no solution, hoping someone here has solved this before, or can point me in the right direction?

**Comment by mfleisz** [comment]
This message means you have no X-server running (or it is misconfigured and is not working correctly). Search to web for more information on how to properly set up a working X environment. This is not a FreeRDP issue.
